src/main/kotlin/com/yapp/itemfinder/sample/service/SampleUserService.kt

```diff
@@ -0,0 +1,42 @@
+package com.yapp.itemfinder.sample.service
+
+import com.yapp.itemfinder.sample.entity.SampleUser
+import com.yapp.itemfinder.sample.repository.SampleUserRepository
+import com.yapp.itemfinder.sample.service.dto.CreateUserReq
+import com.yapp.itemfinder.sample.service.dto.CreateUserRes
+import com.yapp.itemfinder.sample.service.dto.GetUserRes
+import com.yapp.itemfinder.sample.service.dto.UpdateUserReq
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Transactional(readOnly = true)
+@Service
+class SampleUserService(
+    private val userRepository: SampleUserRepository
+) {
+
+    @Transactional
+    fun insertUser(dto: CreateUserReq): CreateUserRes {
+        require(!userRepository.existsByEmail(dto.email)) {IllegalArgumentException("이미 존재하는 회원입니다.")}
```

Comment:
```suggestion
     require(!userRepository.existsByEmail(dto.email)) {
            "이미 존재하는 회원입니다."
        }
```
참고) 요 부분 이렇게 변경할 경우에 require 메서드 내부에서 해당 메시지를 담아서 IllegalArgumentException를 발생시키고 있어용

Reply:
앗 제가 아직 익숙하지 않아서,,, 실수했네요. 감사합니다!

Reply:
지금 이렇게 빨리 코들린에 적응하신 것도 대단합니당 😄